tests/_fixtures/fixturesAuth.ts

```diff
@@ -0,0 +1,76 @@
+import { test as base } from '@playwright/test';
+import { SignUpPage } from "../../src/ui/pages/signUpPage";
+import { SignInPage } from '../../src/ui/pages/signInPage';
+import { HomePage } from '../../src/ui/pages/homePage';
+import { CustomerLookupPage } from '../../src/ui/pages/customerLokupPage';
+import { OpenNewAccountPage } from '../../src/ui/pages/openNewAccountPage';
+import { AccountOverviewPage } from '../../src/ui/pages/accountOverviewPage';
+import { TransferFundsPage } from '../../src/ui/pages/transferFundsPage';
+import { BillPaymentPage } from '../../src/ui/pages/billPaymentPage';
+import { FindTransactionsPage } from '../../src/ui/pages/findTransactionsPage';
+import { ApplyForLoanPage } from '../../src/ui/pages/applyForLoanPage';
+
+export const test = base.extend<{
+  signUpPage;
+  user;
+  homePage;
+  signInPage;
+  customerLookupPage;
+  openNewAccountPage;
+  accountOverviewPage;
+  transferFundsPage;
+  billPaymentPage;
+  findTransactionsPage;
+  applyForLoanPage;
```

Comment:
```suggestion
  customerLookupPage;
  openNewAccountPage;
  accountOverviewPage;
  transferFundsPage;
  billPaymentPage;
  findTransactionsPage;
  applyForLoanPage;
```
a lot of pages here do no relate to auth. Create separate fixture for them and merge it in fixtures.ts file